src/assertions/there.js

```diff
@@ -0,0 +1,21 @@
+import elementExists from '../util/element-exists';
+
+export default function there(client, chai, utils) {
```

Comment:
Yeah, I agree about the `dom` flag.  Maybe we could just come up with a more semantic flag, like... `element` ?  

```javascript
  expect('.some-selector').element.to.exist;
  expect('#other-thing').element.to.have.text('This is so other');
```

Except that doesn't work for `count`... however, we could also have an `elements` flag, which would do the same thing.  And actually, the flag(s) could be optional (only required for things like `exist`).  Other flag options: `$` or `selector`

OR, alternately, we could just think of another name for `there` :)  Like... `existent` - but that just feels so contrived... `there` was the most semantic thing I could think of in practical usage (`expect('thing').to.be.there`).  Other options: `to.show_up` (confused with visible?) or `to.be.found/findable`